src/css/profile/mobile/base.less

```diff
@@ -129,4 +129,6 @@
     --on-off-switch-on-disabled-track-background: @on-off-switch-on-disabled-track-background;
     --on-off-switch-off-track-background: @on-off-switch-off-track-background;
     --on-off-switch-off-disabled-track-border: @on-off-switch-off-disabled-track-border;
+
```

Comment:
Is it correct the profile/mobile profile?
If so, in case of mobile touch , it don't display the focus outline?

Reply:
The focus on mobile profile is disabled.

Focus outline will only work in tv profile, this is enabled in /src/js/profile/tv/config.js
```
	ns.setConfig("keyboardSupport", true);
```